docs/creating-your-site.md

```diff
@@ -44,6 +44,22 @@ theme:
   name: material
 ```
 
+Note that by default, MkDocs will assume that your site is hosted at the root of
```

Comment:
Great work! We should add that the `site_url` setting is strongly recommended from our perspective, as without it, several features will not work, including instant loading and previews, as well as social cards.